tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -28,16 +30,32 @@ class ProduceConsumeValidateTest(Test):
 
     def __init__(self, test_context):
         super(ProduceConsumeValidateTest, self).__init__(test_context=test_context)
+        # How long to wait for the producer to declare itself healthy? This can
+        # be overidden by inheriting classes.
+        self.producer_start_timeout_sec = 60
```

Comment:
Sorry, that is leftover from when I was trying to debug test failures. It doesn't need to be 60 anymore. Will revert to the original 20.